### ISSUE TITLE
Problem: Username on Accounts GQL endpoint not working

### ISSUE THREAD

**Issue by robinbb** [Granola-Team/mina-indexer#1067]

**Comment by trevorbernard** [comment]
> Username always comes back as unknown

Not true. The problem is that we don't have a lot of username data so it appears that it's always Unknown. Closing this issue

<img width="1704" alt="Screenshot 2024-06-11 at 10 04 50 AM" src="https://github.com/Granola-Team/mina-indexer/assets/8825/7fec719e-7f15-4d4f-af1d-3727496ee41f">

**Comment by robinbb** [comment]
On the accounts page, the user for that PK is indeed Unknown. Front end issue? @n1tranquilla

**Comment by n1tranquilla** [comment]
Here are the tests from the FE: https://github.com/Granola-Team/mina-block-explorer/pull/818/files
